### DISCUSSION TITLE
About "HERO RESOURCES AND TECHNICAL ITEM REQUIREMENTS"

### DISCUSSION THREAD

**Discussion by ghost** [ValveSoftware/Dota2-Gameplay#1530] [General]
The 3D models you offer to the artist missing actions, arcana actions, and possess.
That will help a lot with clipping in several normal situations.
Offer more tools to the creators to work with.

In that case if the artist had the arcana possess we will had zero problems but now we have that bad chest. (item that i use is from latest treasure)
![Screenshot (213)](https://user-images.githubusercontent.com/74948827/175500645-2bc51dd2-f434-40b5-b17c-097e954f1d05.png)
![Screenshot (215)](https://user-images.githubusercontent.com/74948827/175500658-47e908e8-0bea-4cab-9431-ae6e0b84a2da.png)
![Screenshot (224)](https://user-images.githubusercontent.com/74948827/175771915-7424ab86-be7b-4a85-aa17-3c651c0be593.png)
![Screenshot (222)](https://user-images.githubusercontent.com/74948827/175771929-a865a2b4-d99b-4b9f-af6e-849ec00d4b33.png)

In that case i use Kinetic Gem "Serene Honor" that change the pose and same thing happen with an other artist cosmetic on legs.
![Screenshot (217)](https://user-images.githubusercontent.com/74948827/175502020-3cb7ebc4-8039-4e64-bf3b-5ca957d896b3.png)
![Screenshot (219)](https://user-images.githubusercontent.com/74948827/175502027-401acfbf-f22a-425e-a35b-08a36e7bed8a.png)

Here we have Juggernaut arcana pose and an expencive cosmetic clipping bad for same reason, artist didnt had enough actions/possess when he was create the models.
![Screenshot (220)](https://user-images.githubusercontent.com/74948827/175503695-57dfb969-7970-4830-82d7-6814b6fc9052.png)

 
I have a solution that i use by my own that contain actions, i can create a sample to share with you, in that case i can offer .blend files with ready basic model and actions  etc. let me know if you want too.

Kind regards.

**Comment by john3342** [comment]
The 3D models you offer to the artist missing actions, arcana actions, and possess.
That will help a lot with clipping in several normal situations[.](crackpatched.com)
Offer more tools to the creators to work with.
[cronacrack](cronacrack.com)
In that case if the artist had the arcana possess we will had zero problems but now we have that bad chest. (item that i use is from latest treasure) [updatedmodapk.com](updatedmodapk.com)
